Sources/SwiftGen/Commander/XCFileListCommands.swift

```
@@ -0,0 +1,94 @@
+//
+// SwiftGen
+// Copyright © 2020 SwiftGen
+// MIT Licence
+//
+
+import Commander
+import PathKit
+import StencilSwiftKit
+import SwiftGenKit
+
+enum XCFileListCLI {
+  private enum CLIOption {
+    static let config = Option<Path>(
+      "config",
+      default: "swiftgen.yml",
+      flag: "c",
+      description: "Path to the configuration file to use",
+      validator: checkPath(type: "config file") { $0.isFile }
+    )
+
+    static let inputs = Option<Path>(
+      "inputs",
+      default: "./swiftgen-input.xcfilelist",
```

I wonder if we should even provide defaults. Pretty sure everyone will have different paths for these. May be better to instead leave these empty, and throw an error if not provided?